source/ios/AdaptiveCards/ADCIOSVisualizer/Card.json

```diff
@@ -0,0 +1,194 @@
+                    [
+                        {
+                            "spacing":"none",
+                            "text":"Pending approval",
+                            "type":"TextBlock",
+                            "wrap":true
+                        },
+                        {
+                            "spacing":"none",
+                            "text":"Daniela Mandera",
+                            "type":"TextBlock",
+                            "wrap":true
+                        },
+                        {
+                            "spacing":"none",
+                            "text":"Daniela Mandera",
+                            "type":"TextBlock",
+                            "wrap":true
+                        },
+                        {
+                            "spacing":"none", 
+                            "text":"I'll be away on assignment in for the next two months and I will need remote access to the network.",
+                            "type":"TextBlock",
+                            "wrap":true
+                        },
+                        {
+                            "spacing":"none", 
+                            "text":"I'll be away on assignment in for the next two months and I will need remote access to the network.",
+                            "type":"TextBlock",
+                            "wrap":true
+                        }
+                    ],
+                    "type":"Column"
+                }
+            ], 
+            "spacing":"default",
+            "type":"ColumnSet"
+        }
+    ], 
+    "type":"AdaptiveCard", 
+    "actions":
+    [
+        {
+            "title":"Set due date",
+            "card":
+            {
+                "type":"AdaptiveCard",
+                "actions":
+                [
+                    {
+                        "title":"OK",
+                        "type":"Action.Submit"
+                    }
+                ],
+                "body":
+                [
+                    {
+                        "id":"dueDate",
+                        "type":"Input.Date"
+                    }
+                ]
+            },
+            "type":"Action.ShowCard"
+        },
+        {
+            "title":"Comment",
+            "card":
+            {
+                "type":"AdaptiveCard",
+                "actions":
+                [
+                    {
+                        "title":"OK",
+                        "type":"Action.Submit"
+                    }
+                ],
+                "body":
+                [
+                    {
+                        "id":"comment",
+                        "placeholder":"Enter your comment",
+                        "type":"Input.Text",
+                        "isMultiline":true
+                    }
+                ]
+            },
+            "type":"Action.ShowCard"
+        }
+    ],
+    "version":"1.0"
+}
+
+    
+
+    
+  
+
+  
+
+  
```

Comment:
we should probably reformat this file... it's a little odd :)

Reply:
is this PR still active?

---
In reply to: [263089207](https://github.com/Microsoft/AdaptiveCards/pull/2505#discussion_r263089207) [](ancestors = 263089207)